File: API/Helpers/CourseParams.cs
namespace API.Helpers
{
    public class CourseParams : PaginationParams
    {
        public string CurrentName { get; set; }
        public string Category { get; set; }
        public string CurrentUsername { get; set; } // del usuario logeado(instructor username)
        public int CurrentUserId { get; set; } // del usuario logeado(instructor id)
        public string OrderBy{ get; set; } = "dateCreated";

    }
}

File: client/src/app/_models/courseParams.ts
import { Course } from "./course";

export class CourseParams {
    category: string;
    pageNumber = 1;
    pageSize =5;
    orderBy = 'dateCreated';

    /*constructor(course: Course) {
        this.category = course.category === 'technology' ? 'cultural' : 'technology'
    }*/
}